app/models/Signup.js

```diff
@@ -129,8 +131,10 @@ signupSchema.statics.lookupCurrent = function (user, campaign) {
       .then(signupDoc => resolve(signupDoc))
       .catch((err) => {
         app.locals.stathatError(statName, err);
```

Comment:
I was looking at the `chatbot` router to refresh my memory on how that Bad Gateway message is getting populated -- we're logging error messages to Stathat [here](https://github.com/DoSomething/gambit/blob/d427963b9c419c6ac571fe4b6d01460c7fb8bc22/app/routes/chatbot.js#L368), so the newly prefixed message should hopefully show up `WhateverIsCallingMe: Bad Gateway`.